src/components/skilltree.jsx

```diff
@@ -235,6 +260,23 @@ function SkillTree() {
                 </div>
 
                 <div className="column">
+                    <div>
+                        <div className="buffs-header">
+							<h3>{t("skills_and_buffs_active_party_buffs")}</h3>
+							<button className="flyff-button" onClick={() => addPartyBuffs()}>{t("skills_and_buffs_add_party")}</button>
```

Comment:
Would be nice to have "Add" and "All" buttons here similar to the RM buffs, so people can still add individual buffs